README.md

```diff
@@ -329,6 +329,23 @@ Events:
   Normal  Started    13m   kubelet            Started container traffic-agent
 ```
 
+#### The Ambassador Agent
+The Ambassador Agent is an optional component of telepresence that gets installed into your cluster by default.
+By creating an account on the Ambassador Cloud, you can then get an actionable view of your cluster for you and your team.
+Users who connect their telepresence installation to the Ambassador Developer Control Plane can create, share and stop intercepts in their clusters, all from the cloud.
+
+##### (Optional) Enable Ambassador Cloud Features
+If the Ambassador Agent has already been setup either standalone or through an Ambassador Edge Stack installation, this step can be skipped.
+
+To enable these features, pass in your Ambassador Cloud Token by running:
+```shell
+kubectl -n ambassador create secret generic ambassador-agent-cloud-token --from-literal=CLOUD_CONNECT_TOKEN=<YOUR AMBASSADOR CLOUD TOKEN>
```

Comment:
I think the secret name should be `traffic-manager-ambassador-agent-cloud-token`